locales/zh-CN/editor.properties

```diff
@@ -125,7 +125,7 @@ CMD_TOGGLE_QUICK_DOCS=快速文档
 
 # Drag and Drop
 
-DND_MAX_FILE_SIZE_EXCEEDED=文件超过支持的最大大小: 3MB
+DND_MAX_FILE_SIZE_EXCEEDED=文件超过支持的最大大小：3MB
```

Comment:
NOTE: if you find that there are files that got accidentally changed in your commit, and you need to fix them to be what they were on `master`:

```
git checkout master locales/zh-CN/editor.properties
git commit --amend --no-edit
```